packages/elements/src/pill/index.ts

```diff
@@ -86,6 +104,25 @@ export class Pill extends ControlElement {
   @property({ type: Boolean, reflect: true })
   private pressed = false;
 
+  /**
+   * Aria indicating state of toggle pill
+   * @ignore
+   */
+  @property({ type: String,
+    reflect: true,
+    attribute: 'aria-pressed',
+    converter: { toAttribute: emptyStringToNull } // TODO: Remove after typescript update to allow nullable for ARIAMixin
+  })
+  public ariaPressed = '';
```

Comment:
Should not override the area attributes because it breaks AOM, you can use JavaScript set/remove attributes instead.

Reply:
I have already updated on the recent commit.